test/unit/argument_parser/detail/format_help_test.cpp

```diff
@@ -105,16 +105,67 @@ TEST(help_add_test, add_option)
     EXPECT_TRUE(ranges::equal((stdout   | ranges::view::remove_if(is_space)),
                                expected | ranges::view::remove_if(is_space)));
 
-    // Tests the --copyright call. TODO: should be uncommented/fixed after --copyright is actually implemented.
-    // const char * argvCopyright[] = {"./copyright", "--copyright"};
-    // argument_parser copyright("copyright", 2, argvCopyright);
-    // copyright.info.long_copyright = "long copyright";
-    // testing::internal::CaptureStdout();
-    // EXPECT_THROW(copyright.parse(), parser_interruption);
-    // stdout = testing::internal::GetCapturedStdout();
-    // expected = "whatever is expected";
-    // EXPECT_TRUE(ranges::equal((stdout   | ranges::view::remove_if(is_space)),
-                               // expected | ranges::view::remove_if(is_space)));
+    // Tests the --copyright call.
+    const char * argvCopyright[] = {"./copyright", "--copyright"};
+    argument_parser copyright("copyright", 2, argvCopyright);
+
+    std::ifstream license_file{std::string{{SEQAN_INCLUDE_DIR}} + "/../LICENSE"};
+    std::string license_string{(std::istreambuf_iterator<char>(license_file)),
+                                std::istreambuf_iterator<char>()};
+    // Test --copyright with empty short and long copyright info.
+    {
+        testing::internal::CaptureStdout();
+        EXPECT_THROW(copyright.parse(), parser_interruption);
+        stdout = testing::internal::GetCapturedStdout();
+        expected = std::string("================================================================================"
+                               "Copyright information for copyright:"
+                               "--------------------------------------------------------------------------------"
+                               "copyright copyright information not available."
+                               "================================================================================"
+                               "This program contains SeqAn3 code licensed under the following terms:"
+                               "--------------------------------------------------------------------------------"
+                               + license_string);
+        EXPECT_TRUE(ranges::equal((stdout   | ranges::view::remove_if(is_space)),
+                                   expected | ranges::view::remove_if(is_space)));
+    }
+    copyright.info.short_copyright = "short copyright\nshortcopyright";
```

Comment:
swap this with line below so the explanation comes first and introduce a newline between the tests. :nail_care:  makes reading easier